app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills += ["JavaScript", "HTML", "CSS"]
```

Comment:
Checklist point 1: Instead of using `+=` operator to add multiple values to the `skills` list, you should use the `extend()` method. This would make your code more aligned with the Python best practices. Here is how you can do it: `self.skills.extend(["JavaScript", "HTML", "CSS"])`